cgmanifest.json

```diff
@@ -504,8 +504,8 @@
       "Component": {
         "Type": "nuget",
         "Nuget": {
-          "Name": "runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography.OpenSsl",
-          "Version": "4.3.0"
+          "Name": "runtime.osx-arm64.runtime.native.System.IO.Ports",
```

Comment:
Looks like this new file somehow snuck into your PR.  I assume it should be removed?  Otherwise changes look good.

Reply:
The cgmanifest test complained and said I needed to update the list